### DISCUSSION TITLE
Cannot find Microsoft Forms 2.0 Object Library in Outlook

### DISCUSSION THREAD

**Discussion by namtrah** [tallguyjenks/Obsidian-For-Business#19] [Q&A] [answered]
Is there a specific version of Outlook you are using? I have Outlook 365 installed and cannot find this in my object library... Just wondering.

**Comment by tallguyjenks** [comment, marked as answer]
the forms library should be available in there, I've yet to see an office version that doesn't have it and we use O365 at my office where i developed this, what are you seeing in your object library?

**Comment by jeremymarx** [comment]
I had the same issue, also on O365. I found that adding a UserForm (right-click the Project, Insert, UserForm), and then removing it, caused VBA to add the forms library for me.
